### PR TITLE
Milstone 1. Bookstore Project components Branch

### PR DESCRIPTION
**Pull request by otmanTR**
In this branch following changes made:
-Installed Reacpt App
-Installed React Router
-Added linters
-Added Routes and Route
-Added Navbar, Book, Form components
-Added Books, Categories pages
-Updated README.md file
-Checked linters

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <>
+
+      <div className="bookList">
+        <h2> List of Books</h2>
+        <ul>
+          <li className="bookItems">
+            <p>{title}</p>
+            <p>{author}</p>
+            <button type="button" className="button key">Remove</button>
+          </li>
```
Comment (amejid) [inline src/components/Books.js]: - [x] Great job structuring your components. :+1: Let's work on some more details to achieve the project requirements. It is suggested that we should utilize a component that takes props to render an individual Book. Kindly extract these lines to a new component and pass the data through props. ✍️

**Project Requirements**
image

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    <>
+      <div className="app">
+        <h1>Bookstore</h1>
+        <nav>
+          <ul className="navContainer">
+            <li className="navItems">
+              <Link to="/">Books</Link>
+            </li>
+            <li>
+              <Link to="/categories">Categories</Link>
```
Comment (amejid) [inline src/components/Navbar.js:14]: - [ ] [OPTIONAL] React router provides a special type of link called `NavLink` which knows whether or not it is `active`. It makes it easier to apply styles on the currently active link. You can check this [documentation](https://reactrouter.com/en/main/components/nav-link) for more insight into the topic. 😎️

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @otmanTR, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Gitflow :heavy_check_mark: 
- Good component structure :heavy_check_mark: 
- React router successfully set up :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navbar.js (hunk 2 above)
Comment (otmanTR) [reply src/components/Navbar.js:14]: Thank you for this good advice:)

File: src/components/Books.js (hunk 1 above)
Comment (otmanTR) [reply src/components/Books.js]: Thank you, it is very useful way :)

**Review by sumairq — APPROVED** [review]
### Hi @otmanTR , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by otmanTR** [comment]
Thank you all :)

**Pull request merged**